tutorials/diagnostics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>I suggest making a remark that you solve the problem and plot the solution.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/813/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>